cerberus/__init__.py

```diff
@@ -8,22 +8,68 @@
 
 """
 
-from __future__ import absolute_import
+from typing import Dict, Optional, Tuple, Union
```

Comment:
In a project locally I added a new rule called "content:" that takes types like "uuid" and the rule maps their processing to python builtins from a string.  It's something I think is missing when using cerberus as a yaml import validation where everything is a string type (and thus you cannot use custom types).

Another thing cerberus is missing is conditional logic, for example it would be nice to be able to say something required only if another field is set to "X".  For example: "required: true if type == 'number'".

Reply:
regarding types, please see #374. there are the *of-rules for conditional logic. they are causing enough trouble, everything that needs to act more distinguished must be solved with customizations.